internal/pkg/utils/testproject/project.go

```diff
@@ -314,6 +320,44 @@ func (p *Project) createBranches(branches []*fixtures.BranchState) error {
 	return nil
 }
 
+func (p *Project) createBucketsTables(buckets []*fixtures.Bucket) error {
```

Comment:
Chyba mi tu nejake logovanie, co sa deje, vid ostatne metody. (`logf`).

Reply:
(Da sa to vyskusat s `TEST_VERBOSE=true`)

Reply:
👍